systems/divine-action.md

```diff
@@ -1,14 +1,54 @@
 # Divine Action
+
 Divine actions are powers granted to characters by divine beings. Divine actions are distinctly different from spells, and are not bound by the same rules.
+
 Divine actions can be obtained by the player through the power of prayer(s).
 
+Divine actions can be split into 2 groups: Miracles and Curses.
+Miracles are usually have positive benefits to the recipient. Curses are usually bestowed upon enemies, and have negative effects.
+
+A player can use their divine actions a number of times per long rest equal to 1 plus their [Zealot](classes.md#zealot) level.
+
+## Orison
+
+An Orison is a divine action that is always ready, and can be used just like any other divine action. By default all [Zealot](classes.md#zealot)s begin with [Minor Respite](#minor-respite). More Orisons can be granted through feats, or other acts (of faith) at the DM's discretion.
 ## Prayers
+
 All [Zealot](classes.md#zealot) characters start with one prayer slot.
+Each prayer has 3 tenets, each providing the character with a divine action.
```

Comment:
I think you need to explain that there is one prayer selected at a time, and when that can be done.